source/lib.ts

```diff
@@ -95,9 +95,9 @@ export class RedisStore implements Store {
 		if ('sendCommand' in options && !('sendCommandCluster' in options)) {
 			// Normal case: wrap the sendCommand function to convert from cluster to regular
 			this.sendCommand = async ({ command }: SendCommandClusterDetails) =>
-				options.sendCommand(...command)
+				options.sendCommand.bind(this)(...command)
```

Comment:
This isn't a huge deal, but could we bind sendCommand & sendCommandCluster once rather than every time they're called?